specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/0.1-preview/confidentialledger.json

```diff
@@ -122,6 +122,37 @@
         }
       }
     },
+    "/app/subLedgerIds": {
+      "get": {
```

Comment:
Make this pageable if there is no limit or impose a limit

Reply:
An array of strings is hard to version. Perhaps return an array of objects: [{id: <subledgerId>, param1: <val>, etc.}]